functions/go/Makefile

```diff
@@ -153,3 +153,6 @@ func-push:
 docs-generate:
 	$(GOBIN)/mdtogo $(CURRENT_FUNCTION) $(CURRENT_FUNCTION)/generated --license=none --strategy=cmdDocs
 
+license:
```

Comment:
I'd suggest moving it closer to check-license: https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/functions/go/Makefile#L102
Same comments for all Makefiles.